README.md

```diff
@@ -53,6 +53,12 @@ The Dockerfile is under the directories in [container](container/)
 
 ## 4. Deployment
 
+Before you deploy the helm chart, you need to setup the kubernetes cluster and install the helm tool. Please refer to the [kubernetes documentation](https://kubernetes.io/docs/setup/) and [helm documentation](https://helm.sh/docs/intro/install/) for more details. We also deliver a quick start script to setup the kubernetes cluster:
+
+```bash
+python3 ./tools/prerequisites/k8s-setup.py
```

Comment:
(For Ubuntu)

Reply:
fixed. change the script with ubuntu tag